include/SFML/Audio/SoundBuffer.hpp

```diff
@@ -127,16 +126,18 @@ class SFML_AUDIO_API SoundBuffer : AlResource
     /// \param sampleCount  Number of samples in the array
     /// \param channelCount Number of channels (1 = mono, 2 = stereo, ...)
     /// \param sampleRate   Sample rate (number of samples to play per second)
+    /// \param channelMap   Map of position in sample frame to sound channel
     ///
     /// \return True if loading succeeded, false if it failed
     ///
     /// \see loadFromFile, loadFromMemory, saveToFile
     ///
     ////////////////////////////////////////////////////////////
-    [[nodiscard]] bool loadFromSamples(const std::int16_t* samples,
-                                       std::uint64_t       sampleCount,
-                                       unsigned int        channelCount,
-                                       unsigned int        sampleRate);
+    [[nodiscard]] bool loadFromSamples(const std::int16_t*              samples,
+                                       std::uint64_t                    sampleCount,
+                                       unsigned int                     channelCount,
+                                       unsigned int                     sampleRate,
+                                       const std::vector<SoundChannel>& channelMap);
```

Comment:
Having both `channelCount` and `channelMap` seems redundant as `channelCount` can be derived from `channelMap.size()`. Same remark applies to other places which have some `channelMap`.

Reply:
Fixed.